linux.md

```diff
@@ -101,25 +113,11 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     cd -
     ```
     This installs Chrome.<br><br>
-12. Install the following Chrome Extensions:
+13. Install the following Chrome Extensions:
     - [React Developer tools Chrome Extension](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
     - [Refined GitHub Chrome Extension](https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf?hl=en)
     - [Socket Security Chrome Extension](https://chrome.google.com/webstore/detail/socket-security/jbcobpbfgkhmjfpjjepkcocalmpkiaop?hl=en)
     - [Web Vitals Chrome Extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
-13. Copy the following text, paste it in the terminal and hit return.<br><br>
-
-    ```bash
-    pnpm create @upleveled/react-app --help
-    ```
-
-    This will prepare a program that we will use in the course. This will take a while and then respond with a message with some usage instructions, similar to this:<br><br>
-    <img src="./general-cra-installed.avif"><br><br>
-
-14. Copy the following text, paste it in the terminal and hit return.<br><br>
-    ```bash
-    pnpm add --global @upleveled/preflight
-    ```
-    This will prepare a program that we will use in the course.<br><br>
 15. <a name="vs-code-settings"></a> Next we will configure VS Code.<br><br>
```

Comment:
looks like the numbering is off by 1 starting with 15

Reply:
fixed numbers [ecb57fe](https://github.com/upleveled/system-setup/pull/91/commits/ecb57fe2e42067fd1de662f9002f6fcaf212d11c)